src/type/schema.js

```diff
@@ -238,6 +238,27 @@ export class GraphQLSchema {
   getDirective(name: string): ?GraphQLDirective {
     return find(this.getDirectives(), directive => directive.name === name);
   }
+
+  toConfig(): {|
+    ...GraphQLSchemaConfig,
```

Comment:
huh this must be new Flow wizardry: this is a nicely concise way of defining types in terms of other types!